app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,880 @@
+    <li>Pick relationship attribute key(s) to represent the related collection. Relationship attribute keys are used to reference the related collection in queries, so pick something that's intuitive and easy to remember.</li>
+    <li>Select desired <a href="/docs/databases-relations#on-delete">on delete</a> behavior.</li>
+    <li>Click the <b>Create</b> button to create the relationship.</li>
+</ol>
+
+<h2><a href="/docs/databases-relations#create-in-code" id="create-in-code">Creating Relationships Programmatically</a></h2>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Node.js</h3>
+        <div class="ide margin-top-small" data-lang="nodejs" data-lang-label="Node.js SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>const { Client, Databases } = require('node-appwrite');
+
+const client = new Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2');               // Your project ID
+
+const databases = new Databases(client);
+
+databases.createRelationshipAttribute(
+    'marvel',     // Database ID
+    'movies',     // Collection ID
+    'reviews',    // Related collection ID
+    'oneToMany',  // Relationship type
+    true,         // Is two-way
+    'reviews',    // Attribute key 
+    'movie',      // Two-way attribute key
+    'cascade'     // On delete action
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>PHP</h3>
+
+        <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP SDK">
+            <pre class="line-numbers"><code class="prism language-php" data-prism>use \Appwrite\Client;
+use \Appwrite\Services\Databases;
+
+$client = (new Client())
+    ->setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    ->setProject('5df5acd0d48c2');               // Your project ID
+
+$databases = new Databases($client);
+
+$databases->createRelationshipAttribute(
+    databaseId: 'marvel',           // Database ID
+    collectionId: 'movies',         // Collection ID
+    relatedCollectionId: 'reviews', // Related collection ID
+    type: 'oneToMany',              // Relationship type
+    twoWay: true,                   // Is two-way
+    key: 'reviews',                 // Attribute key 
+    twoWayKey: 'movie',             // Two-way attribute key
+    onDelete: 'cascade'             // On delete action
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Python</h3>
+        <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
+            <pre class="line-numbers"><code class="prism language-python" data-prism>
+            from appwrite.client import Client
+from appwrite.services.databases import Databases
+
+client = Client() \
+    .set_endpoint('https://[HOSTNAME_OR_IP]/v1') \  # Your API Endpoint
+    .set_project('5df5acd0d48c2')                   # Your project ID
```

Comment:
The backslash + comment doesn't quite work:

<img width="606" alt="image" src="https://user-images.githubusercontent.com/1477010/229948639-7360b946-584c-4710-b3aa-baafc061a275.png">

```suggestion
client = (Client()
    .set_endpoint('https://[HOSTNAME_OR_IP]/v1')     # Your API Endpoint
    .set_project('5df5acd0d48c2'))                   # Your project ID
```